stub/src/main/java/io/grpc/stub/StreamObserver.java

```diff
@@ -39,8 +39,12 @@
  * to the application. For incoming messages, the application implements the {@code StreamObserver}
  * and passes it to the GRPC library for receiving.
  *
- * <p>Implementations are expected to be thread-compatible. Separate {@code StreamObserver}s do
- * not need to be sychronized together; incoming and outgoing directions are independent.
+ * <p>Implementations are expected to be
+ * <a href="http://www.ibm.com/developerworks/library/j-jtp09263/">thread-compatible</a>.
+ * Separate {@code StreamObserver}s do
+ * not need to be synchronized together; incoming and outgoing directions are independent.
+ * Individual {@code StreamObserver}s are not thread-safe: if multiple threads will be
```

Comment:
Saying "not thread-safe" here can be a bit confusing because it is repeating what was previously said; it's unclear if the reader mis-read something earlier. Let's make it clear we are repeating ourselves by adding "since", like "_Since_ individual StreamObservers are not thread-safe, if multiple threads..." (and then swap to comma).